apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/ProjectsResourceTest.java

```diff
@@ -1190,21 +1399,35 @@ void getProjects__whenTraceIsUpdated__thenUpdateProjectsLastUpdatedTraceAt() {
             Project expectedProject = project.toBuilder().id(projectId).lastUpdatedTraceAt(trace.lastUpdatedAt())
                     .build();
 
-            assertAllProjectsHavePersistedLastTraceAt(workspaceId, List.of(expectedProject));
+            assertAllProjectsHavePersistedLastTraceAt(workspaceName, apiKey, List.of(expectedProject));
         }
 
-        private void assertAllProjectsHavePersistedLastTraceAt(String workspaceId, List<Project> expectedProjects) {
-            List<Project> dbProjects = projectService.findByIds(workspaceId, expectedProjects.stream()
-                    .map(Project::id).collect(Collectors.toUnmodifiableSet()));
-
-            for (Project project : expectedProjects) {
-                Awaitility.await().untilAsserted(() -> {
-                    assertThat(dbProjects.stream().filter(dbProject -> dbProject.id().equals(project.id()))
-                            .findFirst().orElseThrow().lastUpdatedTraceAt())
-                            .usingComparator(TestComparators::compareMicroNanoTime)
-                            .isEqualTo(project.lastUpdatedTraceAt());
-                });
-            }
+        private void assertAllProjectsHavePersistedLastTraceAt(String workspaceName, String apiKey,
+                List<Project> expectedProjects) {
+
+            Awaitility.await().untilAsserted(() -> {
+                var actualResponse = client.target(URL_TEMPLATE.formatted(baseURI))
```

Comment:
Please revert this change, or at least the part that requests the projects. This was intentionally written to retrieve the projects directly from the DB and not via the endpoint.
At the moment, the projects' `lastUpdatedTraceAt` does **not** reflect the value persisted in the DB when requested via the endpoint but instead shows a dynamically calculated value. This will be changed in the future when enough projects will have persisted values. Right now, the new assertion doesn't cover what it should.

Reply:
Done in: https://github.com/comet-ml/opik/pull/894/commits/6b27cf897331689c438ae391fdfe4283e3a3f2cb